risc0/binfmt/src/lib.rs

```diff
@@ -49,3 +49,8 @@ pub(crate) const PAGE_WORDS: usize = PAGE_BYTES / WORD_SIZE;
 pub fn compute_image_id(blob: &[u8]) -> Result<Digest> {
     ProgramBinary::decode(blob)?.compute_image_id()
 }
+
+/// Compute and return the kernel image ID of the specified combined user ELF + kernel ELF binary.
```

Comment:
This is only the kernel_id, not combined with user.

Reply:
Right, I wasn't sure how to explain it. The output is a hash of the kernel. The input is the combined bin file